packages/components/NativeRadioButton/macos/RadioButtonViewManager.m

```diff
@@ -0,0 +1,16 @@
+#import <React/RCTComponent.h>
+#import <React/RCTViewManager.h>
+
+@interface RCT_EXTERN_MODULE(RadioButtonViewManager, RCTViewManager)
+
+RCT_EXPORT_VIEW_PROPERTY(title, NSString);
+RCT_EXPORT_VIEW_PROPERTY(enabled, BOOL);
+RCT_REMAP_VIEW_PROPERTY(buttonKey, keyEquivalent, NSString)
+RCT_CUSTOM_VIEW_PROPERTY(state, BOOL, NSButton) {
+	if ([[RCTConvert NSNumber:json] isEqualToNumber:[NSNumber numberWithBool:YES]]) {
```

Comment:
we don't need to  set the value off cases?

Reply:
Button state is off by default, so no need to validate that case, unless I'm missing something?

Reply:
if the state changes and caller asks for state shouldn't it be getting the most up to date value?

Reply:
I think NSButton does that under the hood? Not sure if I'm addressing your question but the state prop allows the client to set the button's initial state, it's not meant to be read on the JS side.

Reply:
I still feel like we should expicitly set the OFF value, in case someone later tries to set this (outside of just as a default value) and is very confused why it won't.